### PR TITLE
Disable creating and updating groups

### PR DESCRIPTION
**Pull request by taimoor-at-opslevel**
## Changelog

- [x] Disable references to update/create in groups
- [x] Change examples featuring groups
- [x] Make a `changie` entry

## Tophatting

### CODE REVIEW

File: opslevel/resource_opslevel_team.go [hunk 1]
```diff
@@ -174,8 +174,8 @@ func resourceTeamCreate(d *schema.ResourceData, client *opslevel.Client) error {
 		ManagerEmail:     d.Get("manager_email").(string),
 		Responsibilities: d.Get("responsibilities").(string),
 	}
-	if group, ok := d.GetOk("group"); ok {
-		input.Group = opslevel.NewIdentifier(group.(string))
```
Comment (taimoor-at-opslevel) [inline opslevel/resource_opslevel_team.go:178]: `input.Group` is removed in opslevel-go so it is necessary to throw errors here too.

**Review by rocktavious — APPROVED** [review]

File: examples/data-sources/opslevel_group/data-source.tf [hunk 2]
```diff
@@ -1,7 +0,0 @@
-data "opslevel_group" "foo" {
```
Comment (davidbloss) [inline examples/data-sources/opslevel_group/data-source.tf:1]: Should this be kept for now?
I wonder if devs migrating their groups to teams might use this data source during that process.

**Review by davidbloss — APPROVED** [review]
Overall looks good! I just had one non-blocking question

File: examples/data-sources/opslevel_group/data-source.tf (hunk 2 above)
Comment (taimoor-at-opslevel) [reply examples/data-sources/opslevel_group/data-source.tf:1]: My thoughts are that the users who are using groups should have been informed as of the last release that groups are going away. So to discourage new users from using groups I think we should get rid of them from examples ASAP.

**Pull request merged**